doc/source/io.rst

```diff
@@ -4515,6 +4517,69 @@ Read from a feather file.
    import os
    os.remove('example.feather')
 
+
+.. _io.parquet:
+
+Parquet
+-------
+
+.. versionadded:: 0.20.0
+
+Parquet provides a sharded binary columnar serialization for data frames. It is designed to make reading and writing data
```

Comment:
It would be nice to have some explanation to users what is the difference between both